Buildings/Fluid/Geothermal/Borefields/TOUGHResponse/BaseClasses/PartialBorefieldWithTough.mo

```diff
@@ -0,0 +1,280 @@
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{-88,54},{-32,-2}},
+          lineColor={0,0,0},
+          fillColor={223,188,190},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{-82,48},{-38,4}},
+          lineColor={0,0,0},
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{-26,54},{30,-2}},
+          lineColor={0,0,0},
+          fillColor={223,188,190},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{-20,48},{24,4}},
+          lineColor={0,0,0},
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{-28,-6},{28,-62}},
+          lineColor={0,0,0},
+          fillColor={223,188,190},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{-22,-12},{22,-56}},
+          lineColor={0,0,0},
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{36,56},{92,0}},
+          lineColor={0,0,0},
+          fillColor={223,188,190},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{42,50},{86,6}},
+          lineColor={0,0,0},
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{38,-4},{94,-60}},
+          lineColor={0,0,0},
+          fillColor={223,188,190},
+          fillPattern=FillPattern.Forward),
+        Ellipse(
+          extent={{44,-10},{88,-54}},
+          lineColor={0,0,0},
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Forward)}),
+    Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,
+            100}})),Documentation(info="<html>
+<p>
+This model simulates a borefield containing one or multiple boreholes
+using the parameters in the <code>borFieDat</code> record.
+</p>
+<p>
+Heat transfer to the soil is modeled using only one borehole heat exchanger
+(To be added in an extended model). The
+fluid mass flow rate into the borehole is divided to reflect the per-borehole
+fluid mass flow rate. The borehole model calculates the dynamics within the
+borehole itself using an axial discretization and a resistance-capacitance
+network for the internal thermal resistances between the individual pipes and
+between each pipe and the borehole wall.
+</p>
+<p>
+The thermal interaction between the borehole wall and the surrounding soil
+is modeled using
+<a href=\"modelica://Buildings.Fluid.Geothermal.Borefields.BaseClasses.HeatTransfer.GroundTemperatureResponse\">
+Buildings.Fluid.Geothermal.Borefields.BaseClasses.HeatTransfer.GroundTemperatureResponse</a>,
+which uses a cell-shifting load aggregation technique to calculate the borehole wall
+temperature after calculating and/or read (from a previous calculation) the borefield's
+thermal response factor.
```

Comment:
What does cell-shifting load aggregation and thermal response factor has to do with TOUGH?